abs.yaml

```
@@ -0,0 +1,2 @@
+upload_channels:
```

Seems to build on 3.13 (on `linux-aarch64` at any rate):

```suggestion
build_env_vars:
  ANACONDA_ROCKET_ENABLE_PY313: yes

upload_channels:
```